docs/source/dev_guide/review_process.rst

```diff
@@ -0,0 +1,76 @@
+.. _review_process:
+
+.. contents::
+   :local:
+
+Review process
+--------------
+
+This short description should serve as a rough guideline on how we deal with
+reviews on pull requests (PR). Some PRs are easy, others are hard. Some require
+multiple changes and continous feedback, others can be merged in flawlessly. So
+this text should communicate our intent and style and not be a go/no-go
```

Comment:
Remove "So", or change to "Therefore". Also, "go/no-go" is unclear; perhaps "instead of being a strict set of rules". Also, instead of "should communicate", maybe "the purpose of this text is to communicate" is stronger.